Pod/Classes/NWSTokenView.swift

```diff
@@ -235,6 +240,8 @@ open class NWSTokenView: UIView, UIScrollViewDelegate, UITextViewDelegate
         {
             self.label.bounds.size = CGSize(width: self.labelMinimumWidth, height: self.labelMinimumHeight)
             self.label.text = labelText
+            self.label.font = self.dataSource?.fontForTokenViewLabel(self) ?? UIFont(name: "HelveticaNeue", size: 14)
```

Comment:
Maybe lets change the fallback font to be a system font for future proofing:

`UIFont.systemFont(ofSize: 14.0)`